packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -284,12 +358,18 @@ class ReorderableListView extends StatefulWidget {
   final Clip clipBehavior;
 
   /// {@macro flutter.widgets.list_view.itemExtent}
+  ///
+  /// Null if this is a [ReorderableListView.separated].
```

Comment:
This should be a full sentence, but also, why? What is someone wanted the same extent for items and separators?

Reply:
It was suggestion of another reviewer 😆 I can try to improve to make it a better sentence.